src/main/java/uk/gov/justice/digital/hmpps/keyworker/services/ReactivateStaff.kt

```diff
@@ -0,0 +1,33 @@
+package uk.gov.justice.digital.hmpps.keyworker.services
+
+import org.springframework.stereotype.Service
+import org.springframework.transaction.annotation.Transactional
+import uk.gov.justice.digital.hmpps.keyworker.domain.ReferenceDataDomain.STAFF_STATUS
+import uk.gov.justice.digital.hmpps.keyworker.domain.ReferenceDataRepository
+import uk.gov.justice.digital.hmpps.keyworker.domain.StaffConfigRepository
+import uk.gov.justice.digital.hmpps.keyworker.domain.getReferenceData
+import uk.gov.justice.digital.hmpps.keyworker.domain.of
+import uk.gov.justice.digital.hmpps.keyworker.model.StaffStatus.ACTIVE
+import java.time.LocalDate
+
+@Transactional
+@Service
+class ReactivateStaff(
+  private val staffConfigRepository: StaffConfigRepository,
+  private val referenceDataRepository: ReferenceDataRepository,
+) {
+  fun returningFromLeave(date: LocalDate) {
+    staffConfigRepository
+      .findAllStaffReturningFromLeave(date.plusDays(1))
```

Comment:
Could maybe change the SQL to use `<= :date` then only send date rather than `date.plusDays(1)`?